nixos/modules/services/networking/syncthing.nix

```diff
@@ -212,8 +212,10 @@ in {
             };
 
             path = mkOption {
-              type = types.str;
-              default = name;
```

Comment:
I think we can keep the default? Removing it makes the example for `services.syncthing.folders` invalid.